platform/Span.h

```diff
@@ -49,29 +49,29 @@ class is_convertible
 
 /**
  * Special value for the Extent parameter of Span.
- * If the type use this value then the size of the array is stored in the object
+ * If the type uses this value, then the size of the array is stored in the object
  * at runtime.
  */
 #define SPAN_DYNAMIC_EXTENT -1
 
 /**
- * Non owning view to a sequence of contiguous elements.
+ * Nonowning view to a sequence of contiguous elements.
  *
  * Spans encapsulate a pointer to a sequence of contiguous elements and its size
  * into a single object. Span can replace the traditional pair of pointer and
- * size arguments passed as array definition in function calls.
+ * size arguments passed as array definitions in function calls.
  *
  * @par Operations
  *
  * Span objects can be copied and assigned like regular value types with the help
- * of copy constructor and copy assignment (=) operator.
+ * of copy constructor and copy assignment (=) operators.
```

Comment:
It is singular in this context; I can rephrase it: "with the help of the copy constructor or the copy assignment (=) operator "